### ISSUE TITLE
Accessibility: Heading missing text

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#288]
I just ran my accessibility checker and it's flagging an h2 header on all pages that is empty. I think it was something to do with the new e-mail form? Here is the line that it's flagging:
![Capture](https://user-images.githubusercontent.com/15264315/54613818-c1fdcf00-4a31-11e9-97a1-acf7130fdaa5.JPG)

**Comment by d4mation** [comment]
I pushed a change to Staging that puts some default text into the Title. :+1:

This was complaining because just before the pop-over is opened, the Title of the pop-over is changed to match what is being shown. But before it is ever opened, it was empty. It is no longer empty.

**Comment by banksashleym** [comment]
Great! This error is gone now, thank you

**Comment by d4mation** [comment]
This change is now Live 👍
